app/policies/concerns/policy_helpers.rb

```diff
@@ -63,12 +63,12 @@ def technical?
   end
 
   # Indicates whether the current user is member of the given organization
-  def member_of?(organization)
+  def member_of? organization
```

Comment:
That's not the convention that's been applied in the codebase so far. I don't mind changing the convention as long as you agree that we'll refactor all existing code as we update it, and that we'll point out all missing parentheses in future reviews.